user/user_validator.go

```diff
@@ -0,0 +1,148 @@
+// Copyright © 2020 Intel Corporation
+//
+// SPDX-License-Identifier: GPL-3.0-only
+
+package user
+
+import (
+	"bytes"
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/clearlinux/clr-installer/cmd"
+	"github.com/clearlinux/clr-installer/log"
+)
+
+// Validator abstracts user validation efforts
+type Validator struct {
+	Login    string
+	UserName string
+	Password string
+}
+
+var (
+	usernameExp  = regexp.MustCompile("^([a-zA-Z]+[0-9a-zA-Z-_ ,'.]*|)$")
+	loginExp     = regexp.MustCompile("^[a-zA-Z]+[0-9a-zA-Z-_.]*$")
+	cracklibpath = "/usr/bin/cracklib-check"
```

Comment:
This should be const as it was in the other file.
Also, for consistency, please rename it to cracklibPath

Reply:
Done